ci-operator/step-registry/ipi/conf/nutanix/dns/ipi-conf-nutanix-dns-commands.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "${BASE_DOMAIN:?BASE_DOMAIN env variable should be defined}" > "${SHARED_DIR}"/basedomain.txt
+
+cluster_name=${NAMESPACE}-${JOB_NAME_HASH}
+base_domain=$(<"${SHARED_DIR}"/basedomain.txt)
+cluster_domain="${cluster_name}.${base_domain}"
+
+export AWS_DEFAULT_REGION=us-west-2  # TODO: Derive this?
```

Comment:
Have you tested this on any other regions?

Reply:
> Have you tested this on any other regions?
Yes, it's working. Just also tested with us-west-1.

Reply:
> Have you tested this on any other regions?
Yes, it's working. Just also tested with us-west-1.